client/my-sites/site-settings/manage-connection/site-ownership.jsx

```diff
@@ -35,16 +39,53 @@ class SiteOwnership extends Component {
 		);
 	}
 
-	renderCurrentUser() {
+	isUserExcludedFromSelector = user => {
 		const { currentUser } = this.props;
+		return (
+			user.linked_user_ID === false ||
+			user.linked_user_ID === currentUser.ID ||
+			! includes( user.roles, 'administrator' )
+		);
+	};
+
+	transformUser( user ) {
+		return { ...user.linked_user_info, ...{ ID: user.ID } };
+	}
+
+	onSelect = user => {
+		const { translate } = this.props;
+
+		accept(
+			translate( 'Are you sure you want to transfer ownership?' ),
+			accepted => {
+				accepted && this.props.changeOwner( this.props.siteId, user.ID, user.name );
+			},
+			translate( 'Yes' ),
+			translate( 'No' ),
```

Comment:
although `Yes` and `No` translate easily enough I wonder if we couldn't help out by being more specific. this is a design question but what does it mean if I'm not sure? If I click "no [I'm not sure]" then I assume the dialog disappears and I'm lost without any further direction or guidance.

what if we used options like, `[ Transfer Ownership ] or [ Keep Ownership ]`